app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -264,11 +264,15 @@ class NavigationOverlayFragment : Fragment() {
         return serviceLocator.fxaRepo.accountState.subscribe {
             when (it) {
                 FxaRepo.AccountState.AUTHENTICATED_WITH_PROFILE -> Unit //TODO
-                FxaRepo.AccountState.AUTHENTICATED_NO_PROFILE ->
+                FxaRepo.AccountState.AUTHENTICATED_NO_PROFILE -> {
                     fxaButton.setImageResource(R.drawable.ic_avatar_authenticated_no_picture)
+                    fxaButton.contentDescription = resources.getString(R.string.fxa_navigation_item_signed_in2)
+                }
                 FxaRepo.AccountState.NEEDS_REAUTHENTICATION,
-                FxaRepo.AccountState.NOT_AUTHENTICATED ->
+                FxaRepo.AccountState.NOT_AUTHENTICATED -> {
                     fxaButton.setImageResource(R.drawable.ic_fxa_login)
+                    fxaButton.contentDescription = resources.getString(R.string.fxa_navigation_item_new)
```

Comment:
This String needs a substitution

Reply:
@mcomella to what? I don't see it in the story

Reply:
If you look at the String definition, it has a placeholder for "Firefox" (which is mentioned in the localization note):
https://searchfox.org/mozilla-mobile/rev/1bc47e1452899910a17191a3c9cd5084dbad0401/firefox-tv/app/src/main/res/values/strings.xml#220